### ISSUE TITLE
Migration documentation incorrect

### ISSUE THREAD

**Issue by michael-wolfenden** [ampedandwired/html-webpack-plugin#237]
The isomorph apps section of the migration document states that you can still use a javascript functions to generate a template by declaring a function with the following signature

```
function(compilationResult, chunks, assets, compilation) {
...
};
```

This is incorrect, the actual signature is:

```
function (templateParams) {
...
}
```

Also, this new function is not a complete replacement for `templateContent` as we no longer have access to the `compilation` object as it not available from `templateParams`. 

The old signature was:

```
function (templateParams, compilation, callback) {
...
}
```

Could the `compilation` property be added to `templateParams`?

**Comment by jantimon** [comment]
Yes sure why not - could you add a pull request with a unit test?

**Comment by michael-wolfenden** [comment]
Any ideas when a new release will be released?

**Comment by jantimon** [comment]
Released as 2.10.0
